src/lib/vclock/vclock.h

```diff
@@ -457,6 +468,11 @@ vclockset_match(vclockset_t *set, const struct vclock *key)
 	return vclockset_first(set);
 }
 
+#define vclockset_foreach(set, vclock) \
+	for ((vclock) = vclockset_first(set); \
+	     (vclock) != NULL; \
+	     (vclock) = vclockset_next((set), (vclock)))
+
```

Comment:
Please, introduce `vclockset_foreach` and `vclock_min/max` as separate commits.

Reply:
Done, also added tests